docs/user-manual-zh.md

```diff
@@ -0,0 +1,316 @@
+# SUDT-Faucets 操作手册
+
+## 一、服务启动准备工作
+
+#### 准备运行环境
+
+1核2G linux 环境即可运行
+
+####  准备2个存有 CKB 的私钥
+
+- 私钥用处：
+
+    - `Owner` 私钥：用来铸造 ，管理` token`。
+    - `Hosted` 私钥：用来发送 `Token` 给邮件接收者
+
+- 申请方法：
+
+    - [MetaMask使用教程](https://zhuanlan.zhihu.com/p/112285438) ：
+
+        - 若是之前没有过CKB 账户私钥
+
+            - 从[metamask.io](https://metamask.io/) 网站安装`Chrome`或`Firefox`浏览器扩展，可以生成账户，导出私钥
+
+        - 若是之前有CKB 账户私钥
+
+            - 安装 [metamask](https://metamask.io/)   ,选择导入`Owner` 私钥，记录下其 `ETH` 地址
+
+
+
+#### 准备SendGrid账号
+
+- 账号用处：该系统使用了[SendGrid](https://sendgrid.com/) 代理批量发送邮件，所以需要提供 [SendGrid](https://sendgrid.com/) 账号
+
+- 申请方法：注册一个[SendGrid](https://sendgrid.com/) 账号，获取一个[API_KEY](https://docs.sendgrid.com/ui/account-and-settings/api-keys) ，并进行[认证](https://docs.sendgrid.com/ui/account-and-settings/how-to-set-up-domain-authentication)
+
+#### 准备域名
+
+- 若是通过域名对外访问，可自行准备域名，
+
+- 若是通过 IP 对外访问
+
+
+
+#### 章末小结
+
+若是完成了上述准备工作，你应该可以获得类似下面的数据，此处数据均是样例
+
+```shell
+# 私钥信息，将你生成的私钥导入 metamask, 请仔细保管私钥，涉及资金安全，极为重要‼️。
+Owner 私钥： 0x... 
+Owner 地址： 0xxxx
+Hosted 私钥：0x... 
+
+# SendGrid 邮件信息
+SENDGRID_API_KEY='SB.2BiBXG1gQvqr9BfqdkUlSB.2B43jMWnccTmCNHOllfKzAAVjteyduViTHTZrE0UjSB'
+SENDGRID_VERIFIED_SENDER='xxxxx@gmail.com'
+
+# 域名信息
+https://sudt.faucet.me/
+```
+
+
+
+## 二、构建启动服务
+
+#### 准备环境
+
+- 系统信息
+
+- 环境依赖
+
+    - Docker： docker version
+
+      ```shell
+      Client: Docker Engine - Community
+       Version:           19.03.12
+      
+      Server: Docker Engine - Community
+       Engine:
+        Version:          19.03.12
+      ```
+
+    - Node：v12.18.2 及以上
+
+#### 依赖构建
+
+```shell
+# 下载 submodule
+cd sudt-faucet && rm -rf ckit && git clone https://github.com/homura/ckit.git
+
+# 切换 submodule 分支，并且构建 submodule 代码
+cd ckit && git checkout develop && git pull && cd ../ && yarn install && yarn build:lib 
+
+# 构建 app-ui-issuer 代码
+cd packages/app-ui-issuer  && yarn build && cd ../../
+
+# 构建 app-server-issuer 代码。
+cd packages/app-server-issuer &&  yarn build && cd ../../
+
+# 构建 app-ui-claim 代码
+cd packages/app-ui-claim && yarn build && cd ../../
```

Comment:
we should expose env before building the `app-ui-x`, since `create-react-app` require the env variables when building.

```
# read .env file and expose them as current env
export $(grep -v '^#' .env.lina | xargs)

yarn workspace @sudt-faucet/app-ui-issuer run build
```